src/gsAssembler/gsExprEvaluator.h

```diff
@@ -18,7 +18,7 @@
 #include <gsAssembler/gsQuadrature.h>
 #include <gsAssembler/gsRemapInterface.h>
 #include <gsAssembler/gsCPPInterface.h>
-//#include <gsIO/gsWriteParaview.h>
+#include <gsIO/gsWriteParaview.h>
```

Comment:
this include creates a problem in header-only builds.
Can we do without it?

Reply:
When using `writeParaview` of `gsExprEvaluator`, it gives a compiler error:
```
.../src/gsAssembler/gsExprEvaluator.h:929:34: error: ‘gsWriteParaviewTPgrid’ was not declared in this scope
  929 |             gsWriteParaviewTPgrid( gmap ? pts : pt.toMatrix(), // parameters
      |             ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  930 |                                   vals,
      |                                   ~~~~~
  931 |                                   pt.numPointsCwise(), fileName );
```
So it needs to find `gsWriteParaviewTPgrid`.

Reply:
Hm, I checked again on gcc and it still works. If it does not, instead of including gsWriteParaview.h we should just forward declare gsWriteParaviewTPgrid

Reply:
OK, in 30e6e4e175273190a868721cced618d70fe4b5c6 I move the forward declarations, it is not perfect but it should fix the issue.